src/software/gui/geometry_conversion.h

```diff
@@ -36,3 +36,13 @@ QPolygonF createQPolygonF(const Polygon& polygon);
  * @return The QLineF representation of the given Segment
  */
 QLineF createQLineF(const Segment& segment);
+
+/**
+ * Converts Qt's Point class to our Point class
+ *
+ * @param point The Point to convert
+ *
+ * @return The Point representation of the given QPoint
+ */
+Point createPoint(const QPointF& point);
```

Comment:
I feel like convertPoint() would suit these functions better

Reply:
This is currently consistent with the rest of this file. If we want to discuss renaming all of them please make a ticket (and link it here).

Reply:
I'd maybe prefer `convertToQPoint`, but I agree with @MathewMacDougall that it's out-of-scope for this PR.